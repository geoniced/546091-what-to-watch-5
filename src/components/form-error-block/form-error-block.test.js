import React from "react";
import renderer from "react-test-renderer";
import FormErrorBlock from "./form-error-block";

describe(`FormErrorBlock render`, () => {
  it(`renders FormErrorBlock component when there is no error`, () => {
    const formErrorsMock = {};

    const tree = renderer
      .create(
          <FormErrorBlock error={formErrorsMock.someError}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders FormErrorBlock component when there is error`, () => {
    const formErrorsMock = {
      someError: `There is an error`,
    };

    const tree = renderer
      .create(
          <FormErrorBlock error={formErrorsMock.someError}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
