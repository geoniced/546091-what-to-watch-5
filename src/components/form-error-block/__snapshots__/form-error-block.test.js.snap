// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormErrorBlock render renders FormErrorBlock component when there is error 1`] = `
<div
  style={
    Object {
      "color": "red",
      "padding": "10px 20px",
      "textAlign": "center",
    }
  }
>
  There is an error
</div>
`;

exports[`FormErrorBlock render renders FormErrorBlock component when there is no error 1`] = `null`;
